test/commands/scanner/run.filters.test.ts

```diff
@@ -44,8 +44,8 @@ describe('scanner:run tests that result in the use of RuleFilters', function ()
 
 	describe('--category and --engine flag', () => {
 		describe('Eslint Javascript Engine --category flag', () => {
-			const category = 'Possible Errors';
-			const expectedViolationCount = 4;
+			const category = 'problem';
```

Comment:
The removal of old-school categories means that these tests needed to use the new types. This changed the expected violation count. Tried to make it so each test uses a different type, the way the old tests used different categories.